BackEnd/Functions/ImageModification/ColorChanging.py

```diff
@@ -51,15 +51,19 @@ def change_colors(self, parameters: list[dict]) -> None:
             for y in range(height):
                 r, g, b = self._pixels[x, y]
                 for parameter in parameters:
-                    start_colors = [int(num) for num in parameter['ColorRange']['FromColor'].split(',')]
-                    end_colors = [int(num) for num in parameter['ColorRange']['ToColor'].split(',')]
-                    to_colors = (int(num) for num in parameter['ReplaceColor'].split(','))
-                    r_start, g_start, b_start = start_colors
-                    r_end, g_end, b_end = end_colors
+                    r_start = parameter['ColorRange']['FromColor']['r']
```

Comment:
Can keep 'parameter['ColorRange']['FromColor']' in a variable for shorter and more readable syntax